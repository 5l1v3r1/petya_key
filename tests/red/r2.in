e4Pb8YnLu3LBLViHMPcSFx5A4rEW7RcuYdVjiMBqccTbQosDqZvSvrVZxnn9w1J5bGk2JyQzV39Kb47newj1LQ6Qfp
