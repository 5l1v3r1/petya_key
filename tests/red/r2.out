8fb9GLT7qkQJ5hBu
